pkgs/top-level/all-packages.nix

```diff
@@ -42338,4 +42338,6 @@ with pkgs;
   ssl-proxy = callPackage ../tools/networking/ssl-proxy { };
 
   code-maat = callPackage ../development/tools/code-maat {};
+
+  arrpc = callPackage ../applications/networking/instant-messengers/arrpc {};
```

Comment:
How about using `pkgs/by-name/`?

Reply:
Done. Although I am not sure why this part is not marked "Outdated." Especially since this change is visible in "Commits" and "Files Changed"